src/core_modules/capture-core/components/InfoIconText/InfoIconText.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
![image](https://user-images.githubusercontent.com/4181674/101020174-ff87f000-3565-11eb-9ae3-fe125802fdcf.png)
I am creating this component since I find myself reusing it